apps/ideaspace/src/components/modules/EditIdea/EditIdea.js

```diff
@@ -82,15 +77,17 @@ function EditIdea() {
 
   useEffect(async () => {
     if (ideaId) {
-      const idea = cleanData(await agent.Ideas.getIdea(ideaId, new URLSearchParams("populate=*")));
+      const idea = cleanData(
+        await agent.Ideas.getIdea(ideaId, new URLSearchParams('populate=*'))
```

Comment:
@becevka we should revisit how many collections are being queried with `populate=*`.

Reply:
> @becevka we should revisit how many collections are being queried with `populate=*`.

@chungthuang @becevka how would we access info on the author of the idea without populating at least one "level"?

Reply:
I think you can get it from `populate=author`. But I didn't mean to block this PR, just something we were discussing in the backend team about the complexity of the relations between our collections.

Reply:
> I think you can get it from `populate=author`. But I didn't mean to block this PR, just something we were discussing in the backend team about the complexity of the relations between our collections.

That's good to know. Thanks!